src/main/java/org/folio/oaipmh/helpers/AbstractGetRecordsHelper.java

```diff
@@ -252,7 +252,12 @@ protected void requestAndProcessSrsRecords(Request request, Context ctx, Promise
         .reduce((resultLocal, resultCentral) -> {
           JsonArray sourceRecordsLocal = resultLocal.getJsonArray(SOURCE_RECORDS);
           JsonArray sourceRecordsCentral = resultCentral.getJsonArray(SOURCE_RECORDS);
+          logger.info("Number of SRS records from central tenant: {}", sourceRecordsCentral.size());
```

Comment:
Looks you can join both log messages in one string and use only one logger.info() operation.